InfiniteRecharge/src/main/java/frc/robot/subsystem/scoring/shooter/ShooterSubsystem.java

```diff
@@ -278,12 +304,14 @@ public void rotate(double spinRateAzimuth, double spinRateElevation) {
     }
 
     public void rotateToDeg(double targetPointAzimuth, double targetPointElevation) {
+        // Calculate the target position of the turret in ticks.
         double targetPointTicksAzimuth = MathUtils.unitConverter(targetPointAzimuth, 360,
                 config.shooter.azimuth.ticksPerRevolution) / config.shooter.azimuthGearRatio;
 
         double targetPointTicksElevation = MathUtils.unitConverter(targetPointElevation, 360,
                 config.shooter.elevation.ticksPerRevolution) / config.shooter.elevationGearRatio;
 
+        // Set the target position to the calculated number.

```

Comment:
Can we also tell the reader that the turret motor will be commanded to this position in periodic()?